client/cypress/e2e/_16allocationAWOpen.cy.ts

```diff
@@ -0,0 +1,332 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import chaiColors from 'chai-colors';
+
+import { connectWallet, mockCoinPricesServer, visitWithLoader } from 'cypress/utils/e2e';
+import { moveTime, setupAndMoveToPlayground } from 'cypress/utils/moveTime';
+import viewports from 'cypress/utils/viewports';
+import { QUERY_KEYS } from 'src/api/queryKeys';
+import {
+  HAS_ONBOARDING_BEEN_CLOSED,
+  IS_ONBOARDING_ALWAYS_VISIBLE,
+  IS_ONBOARDING_DONE,
+} from 'src/constants/localStorageKeys';
+import { ROOT_ROUTES } from 'src/routes/RootRoutes/routes';
+
+chai.use(chaiColors);
+
+Object.values(viewports).forEach(
+  ({ device, viewportWidth, viewportHeight, isLargeDesktop, isDesktop }) => {
+    describe(`[AW IS OPEN] Allocation: ${device}`, { viewportHeight, viewportWidth }, () => {
+      before(() => {
+        /**
+         * Global Metamask setup done by Synpress is not always done.
+         * Since Synpress needs to have valid provider to fetch the data from contracts,
+         * setupMetamask is required in each test suite.
+         */
+        cy.setupMetamask();
+      });
+
+      beforeEach(() => {
+        mockCoinPricesServer();
+        localStorage.setItem(IS_ONBOARDING_ALWAYS_VISIBLE, 'false');
+        localStorage.setItem(IS_ONBOARDING_DONE, 'true');
+        localStorage.setItem(HAS_ONBOARDING_BEEN_CLOSED, 'true');
+        visitWithLoader(ROOT_ROUTES.home.absolute);
+      });
+
+      it('User with UQ score below 15 can allocate rewards to personal and to projects', () => {
```

Comment:
User can always allocate to personal and to projects, regardless of the UQ. Is there any specific case here?

Reply:
I wanted to check allocation together with Low UQ Modal.

Reply:
So I suggest adding it to the description. :).

```suggestion
      it('User with UQ score below 15 can allocate rewards to personal and to projects, ModalAllocationLowUqScore is shown to them', () => {
```